### ISSUE TITLE
Datashader object does not update in Holoviews after update to 0.6.1

### ISSUE THREAD

**Issue by DanielKress** [bokeh/datashader#461]
Dear datashade community,

I wrote some functions to visualize large time series dataframes using holoviews and datashader with bokeh extension enabled. Here are the main elements of the function to give you an idea of how I implemented datashader:

```
import holoviews as hv
from holoviews.operation.datashader import datashade

point = hv.Scatter(df)
shade = datashade(point, cmap = colormap).opts(plot = dict_ds)
layout = hv.Layout(shade).opts(plot = dict_lo).cols(1)
return layout
```

Basically, the these are the core elements you use in your example notebooks. 
For adjusting plot options, I am using dicts with .opts and not cell magic as this turned out to be a pain when transferring the code from jupyter to python scripts as need when creating convinient plotting modules.

Today, I updated datashader from 0.6 to 0.6.1 which resulted in crashs of my plot functions (did not save the output, sorry...). Therefore, I made sure that I had holoviews in the most recent version, which it was... Thereafter, I updated bokeh from 0.12.7 to 0.12.9. The result was that the function works again but creates interactive plots with a single static datashade rendered image. A zoom in does not cause an update in the rendering...

![image](https://user-images.githubusercontent.com/6912630/30375340-74f9dfca-9888-11e7-9c1e-9a481560993d.png)

My question, did you guys change something in the way the server updates the data when the plot gets interacted with? How can I make use of the great datashader module again?

Thx for your help!!!

**Comment by philippjfr** [comment]
There were some changes in bokeh that caused this problem. We'll try to get a hotfix release of holoviews out later today or tomorrow. I do worry about your initial issue though:

>I updated datashader from 0.6 to 0.6.1 which resulted in crashs of my plot functions (did not save the output, sorry...).

Do you have some way to reproduce this?

**Comment by DanielKress** [comment]
Thx for the quick reply! Great to hear that holoviews is getting an hotfix!!! 

As fix, I downgraded my modules (bokeh > 0.12.7 + datashader >0.6.0) and the plotting function worked again.

To reproduce the error, I updated only datashader 0.6.1 again: 

![image](https://user-images.githubusercontent.com/6912630/30378941-2c611492-9895-11e7-8593-161bda9a862a.png)

The function is still working... So, I am not able to reproduce the issue. Maybe the kernel was in a bad state last time. Thanks anyway
